src/lib/protocols/ftp_control.c

```diff
@@ -64,6 +64,11 @@ static int ndpi_ftp_control_check_request(struct ndpi_detection_module_struct *n
     return 1;
   }
 
+  if(ndpi_match_strprefix(payload, payload_len, "AUTH") ||
```

Comment:
`ndpi_match_strprefix` calls `ndpi_match_prefix` which call `memcmp`, unsure if `strncasecmp`would be faster and if strings like `aUtH` are also valid